.github/workflows/ci.yml

```diff
@@ -31,7 +31,6 @@ jobs:
         # NB(gcasassaez): TF 2.8 moves numpy to >=1.20 wich doesnt work for pyarrow<3
         # See: https://issues.apache.org/jira/browse/ARROW-11450
         depconstraint: [
-          "tfx~=1.0.0 tensorflow~=2.5.0"
```

Comment:
why are we removing this btw?

Reply:
I thought we only support between min and max tfx versions with 1.4.0 as min.

Reply:
true. Fixed in https://github.com/tensorflow/tfx-addons/pull/119. Seems the file had an typing error